microsoft/BotFramework-WebChat
# Fix Suggested Actions accessibility

> Fixes #2713.

## Changelog Entry

### Fixed

-  Fixes [#2713](https://github.com/microsoft/BotFramework-WebChat/issues/2713). Suggested actions container should persist for AT, by [@corinagum](https://github.com/corinagum) in PR [#2712](https://github.com/microsoft/BotFramework-WebChat/pull/2712)

## Description

Narration of suggested actions is done via `aria-live="polite"`.

Today, the suggested actions container does not exists when the suggested actions are received, thus, AT do not narrate the suggested actions because AT is designed not to narrate new content in the `aria-live="polite"` container. It only narrate changed contents.

## Specific Changes

- Modify `SuggestedActions.js` to show a persisted container even there are no suggested actions.

---

-  [x] ~Testing Added~ (Will manual test)
   1. Turn on AT
   1. Type "suggested-actions"
   1. Expect: AT should narrate the content of suggested actions
